examples/mysql-database-example.yml

```diff
@@ -5,17 +5,57 @@ integrations:
   - name: nri-flex
     # interval: 30s
     config:
-      name: mysqlDbFlex
+      name: MySQLDbFlex
       apis:
         - database: mysql
-          db_conn: hithere:byethere@tcp(apacmysql.adcasdasdfa.ap-southeast-2.rds.amazonaws.com:3306)/sys
+          # connection string for an rds mysql instance. Be sure to use the writer endpoint here.  /sys is the default database. newrelic:Password is the user:pass.
+          db_conn: newrelic:Password@tcp(rds-name.region.rds.amazonaws.com:3306)/sys 
           logging:
             open: true
           custom_attributes: # applies to all queries
-            host: myDbServer
+            host: AuroraMySQL # custom host name atribute
           db_async: true # process queries async
           db_queries:
-            - name: mysqlSessionSample
-              run: Select * from sys.dm_exec_sessions
+            - name: MySQLShowVars
+              run: SHOW VARIABLES;
               custom_attributes: # can apply additional at a nested level
-                nestedAttr: nestedVal
+                nestedAttr: max-conns
+            - name: MySQLCustomQuery
```

Comment:
should this be named something more relative to the query similar to the examples on lines 19 and 27?

Reply:
Good point. I'll update the names to be more consistent.